corehq/apps/geospatial/static/geospatial/js/models.js

```diff
@@ -660,6 +660,12 @@ hqDefine('geospatial/js/models', [
                     alertUser.alert_user(message, 'success');
                     window.location.reload();
                 },
+                error: function () {
+                    alertUser.alert_user(
+                        gettext("Oops! Something went wrong! Please contact admin if the problem persists."),
```

Comment:
what is the usual message we have on HQ? is it contact admin or report an issue?

Reply:
report an issue is the usual message and is more appropriate as well for this use case. Addressed in 02da8893ae2bed12b955dadc0c477feafe63d105